sklearn/utils/tests/test_extmath.py

```diff
@@ -330,6 +331,39 @@ def test_randomized_svd_sign_flip():
         assert_almost_equal(np.dot(v2.T, v2), np.eye(2))
 
 
+def test_randomized_svd_sign_flip_with_transpose():
+    # Check if randomized_svd flipping is always done based on u
+    # irrespective of transpose.
+    # See https://github.com/scikit-learn/scikit-learn/issues/5608
+    # for more details.
+    def max_loading_is_positive(u, v):
+        """
+        returns bool tuple indicating if
+        the values maximising np.abs
+        are positive across all rows
+        for u and across all columns for v.
```

Comment:
Style could you please avoid splitting too much: please break around 80 columns, not 30.